### ISSUE TITLE
Can you add an option to generate the .env file if it doesn't already exist?

### ISSUE THREAD

**Issue by benalman-toast** [bkeepers/update-dotenv#2]
(that would be very nice)

Thanks!

**Comment by OwenMelbz** [comment]
@benalman-toast I believe it already does, on OSX anyway.

It just does `await promisify(fs.writeFile)(filename, contents)`

**Comment by cowboy** [comment]
Yeah. That's totally bizarre, I have no idea why this was an issue for me. I don't have access to that old account, so I can't close the issue. @bkeepers can you close it? Thanks!
